locale/en/locale.cfg

```diff
@@ -121,79 +127,85 @@ ore_richness_normal=Normal
 upgrade_filter=Pollution Filter
-upgrade_filter_message=Comfylatron: Train's pollution filter was upgraded.
-upgrade_filter_tooltip=Train Pollution Filter. Decreases pollution from inside of train and from chrono engine.\nCurrent pollution made by train: __1__%
+upgrade_filter_message=Comfylatron: Chronotrain's pollution filter upgraded.
+upgrade_filter_tooltip=Train Pollution Filter. Dampens pollution from machines inside the train and from charging the chrono engine.\nCurrent machine pollution transfer factor: __1__%
 upgrade_accumulators=Accumulators
-upgrade_accumulators_message=Comfylatron: Train's accumulator capacity was upgraded.
-upgrade_accumulators_tooltip=Add additional row of Accumulators for train's charging system.\nEach accumulator adds 300kW possible power input.
+upgrade_accumulators_message=Comfylatron: Chronotrain's accumulator capacity upgraded.
+upgrade_accumulators_tooltip=Add an additional row of accumulators to the Chronotrain, increasing the maximum power the chrono engine can draw.
 upgrade_loot_pickup=Loot Pickup Range
-upgrade_loot_pickup_message=Comfylatron: Players now have additional red inserter installed on shoulders, increasing their item pickup range.
+upgrade_loot_pickup_message=Comfylatron: Players install additional red inserters on their shoulders. Item pickup range increased.
 upgrade_loot_pickup_tooltip=Add loot pickup distance to players. Current: +__1__ tiles range
 upgrade_inventory_size=Character Inventory Size
-upgrade_inventory_size_message=Comfylatron: Players now can carry more trash in their unsorted inventories.
+upgrade_inventory_size_message=Comfylatron: Players can now carry more trash in their unsorted inventories. Just what we needed uwu
 upgrade_inventory_size_tooltip=Add +10 inventory slots to all players.
 upgrade_repair=Train Repair Speed
-upgrade_repair_message=Comfylatron: Train now gets repaired with additional repair kit at once.
-upgrade_repair_tooltip=Train uses more repair tools at once from Repair chest. Current: +__1__
+upgrade_repair_message=Comfylatron: The Chronotrain can now repair with an additional repair kit at once.
+upgrade_repair_tooltip=The Chronotrain uses more repair tools at once from the Repair Chest. Current: +__1__
 upgrade_water=Piping System
-upgrade_water_message=Comfylatron: Train now has piping system for additional water sources.
+upgrade_water_message=Comfylatron: The Chronotrain now has a piping system for additional water sources in each wagon. Am I allowed in the pool party??
 upgrade_water_tooltip=Add piping through wagon sides to create water sources for each wagon.
 upgrade_output=Output System
-upgrade_output_message=Comfylatron: Train now has output chests.
-upgrade_output_tooltip=Add comfylatron chests that output outside (into cargo wagon 2 and 3)
+upgrade_output_message=Comfylatron: The Chronotrain has been upgraded with output chests. Now it goes both ways!
```

Comment:
```suggestion
upgrade_output_message=Comfylatron: The Chronotrain has been upgraded with output chests.
```

Reply:
IMO original language is unclear, need to give some hint of what 'output chests' mean to players who don't know the scenario